app/assets/javascripts/psique/documentation.js.coffee

```diff
@@ -0,0 +1,8 @@
+# ------------------------------
+# Smooth Scrolling for Styleguide
+# ------------------------------
+$(document).ready ->
+  $('a').click ->
```

Comment:
All the links will be animated?  :-/

I usually prefer to keep the standard behavior when no class is specified and use a class to add customized behaviors...